### ISSUE TITLE
Running cachebust multiple times is appending multiple hash attribute 

### ISSUE THREAD

**Issue by ashokdudhade** [furzeface/gulp-cache-bust#4]
I'm running cachebust on my html files multiple times and it is appending hash multiple times.

```
 <link rel="stylesheet" href="/css/app.css?hash=6b105199b616f251513cf2141de44e70?hash=f4e6f846d936bf416aa3202f941ffb50">
```

**Comment by ycswaves** [comment]
A temporary solution is to add [this package](https://www.npmjs.com/package/gulp-replace), add remove your appended query before you apply the new cache bust.
```
gulp.task('cacheBust', function () {    
    return gulp.src('dist/*.html')
          .pipe(Replacer(/\?t=\d+/g, '')) //look for "?t=1474253123732"
          .pipe(cacheBuster({
            type: 'timestamp'
          }))
          .pipe(gulp.dest('dist'));    
});
```

**Comment by davinkevin** [comment]
Seems to be fixed in v1.1.0, could you confirm ?

**Comment by davinkevin** [comment]
No confirmation of this, I close the issue.
